src/plugins/data/public/autocomplete/providers/value_suggestion_provider.ts

```diff
@@ -15,13 +15,16 @@ import { AutocompleteUsageCollector } from '../collectors';
 
 export type ValueSuggestionsGetFn = (args: ValueSuggestionsGetFnArgs) => Promise<any[]>;
 
+export type Method = 'terms_enum' | 'terms_agg';
```

Comment:
👍   Can we maybe prefix with `ValueSuggestions` so that when it's used outside of the autocomplete context it is more descriptive?

Reply:
Can we also move this export to src/plugins/data/common/constants.ts and replace the usages of the string in src/plugins/data/server/autocomplete/value_suggestions_route.ts? Thanks for cleaning this up!